### ISSUE TITLE
Github API deprecated

### ISSUE THREAD

**Issue by Ghost-chu** [BookStackApp/BookStack#2010]
**Describe the bug**
Github sent a notice to my email say BookStack using Github API was deprecated.

**Steps To Reproduce**
1. Use Github oAuth to log in BSAPP.
**Expected behavior**
```
On April 7th, 2020 at 03:53 (UTC) your application ([delete]) used an access token (with the User-Agent GuzzleHttp/6.5.1 curl/7.29.0 PHP/7.3.16) as part of a query parameter to access an endpoint through the GitHub API:

https://api.github.com/user

Please use the Authorization HTTP header instead as using the `access_token` query parameter is deprecated.

Depending on your API usage, we'll be sending you this email reminder on a monthly basis.
```
**Screenshots**

**Your Configuration (please complete the following information):**
 - Exact BookStack Version (Found in settings): 0.28.2
 - PHP Version: above
 - Hosting Method (Nginx/Apache/Docker): nginx

**Additional context**

**Comment by ssddanbrown** [comment]
Thank you for reporting this @Ghost-chu,

This has already been addressed under #1879 and the fix is ready to be part of the next release when it deploys.

**Comment by McBochi** [comment]
This should be a known (and fixed) issue with Laravel Socialite: https://github.com/laravel/socialite/pull/430
According to the changelog, this adaption got included into v4.3.2: https://github.com/laravel/socialite/blob/4bd66ee416fea04398dee5b8c32d65719a075db4/CHANGELOG.md
And this dependency was also updated for this project: https://github.com/BookStackApp/BookStack/commit/b0610d85dac280aa36a9e76cca6df8230edd1a14

I guess a simple pull / update of your application should do. :)
